indicoio/api/prebuilt.py

```diff
@@ -4,13 +4,35 @@
 
 from indicoio.preprocess.pdf import pdf_preprocess
 from indicoio.errors import IndicoInputError
+from indicoio.client.storage import StorageClient
 from typing import List
 
 
 def _convert_options_to_str(options):
     return ",".join(f"{key}: {json.dumps(option)}" for key, option in options.items())
 
 
+def enforce_list(inputs):
+    if not isinstance(inputs, list):
```

Comment:
We normally wrap it for them. and unwrap accordingly

Reply:
fixed